index.js

```diff
@@ -65,15 +108,15 @@ NodeWebkitBrowser.prototype = {
   name: 'NW.js',
 
   DEFAULT_CMD: {
-    linux: findpath(),
-    darwin: findpath(),
-    win32: findpath()
+    linux: null,
```

Comment:
Would it be better to have an empty object (with appropriate docs explaining what's going on)?

Reply:
I presume this is conforming to a contract with the superclass which `_getCommand` uses, so I prefer to leave it. I think `null` is a better indicator than `undefined`, as it demonstrates it has deliberately fallen back to this default, whereas `undefined` could result from numerous additional unintended scenarios.

Reply:
Should we add any docs then?

Reply:
Not sure what I'd put that isn't either a bit of a guess, or obvious from a quick inspection of the rest of the code in the file. IMHO it's OK as it is.